packages/dashboards/src/lib/components/timeseries-widget/chart-presets/xy-chart/xy-chart.component.spec.ts

```diff
@@ -166,5 +166,23 @@ describe("XYChartComponent", () => {
             });
             expect(spy).toHaveBeenCalledWith({ payload: timeframe });
         });
+
+        it("should return true if the series is set to interactive", () => {
+            component.configuration = { interaction: "series" } as ITimeseriesWidgetConfig;
+            component.ngOnChanges(initializationChanges);
+            expect(component.isSeriesInteractive(component.chartAssist.legendSeriesSet[0])).toBe(true);
+        });
+
+        it("should communitcate interaction on the EventBus when series is interactive", () => {
+            const spy = spyOn(eventBus.getStream(INTERACTION), "next").and.callThrough();
+            component.configuration = { interaction: "series" } as ITimeseriesWidgetConfig;
+            component.ngOnChanges(initializationChanges);
+            component.onInteraction(component.chartAssist.legendSeriesSet[0]);
+            const interactionData = {
+                payload: { data: component.chartAssist.legendSeriesSet[0], interactionType: TimeseriesInteractionType.Series },
+                id: "INTERACTION",
+            };
+            expect(spy).toHaveBeenCalledWith(interactionData);
+        });
```

Comment:
I don't thing the interaction event should be fired if it's a simple `<a>` tag. Correct @lukasholcik? If not, we should probably have tests that check to make sure the event isn't emitted in case the primary and/or secondary descriptions are just simple links with `<a>` tags.

Reply:
It looks like we still need tests to ensure that the event isn't emitted when it's a simple link for both the primary and secondary descriptions. And, maybe a test for ensuring that it's not emitted when there's neither a simple link nor a data source-provided interaction.